flag provided but not defined: -ingress-template-file

```
2016/09/27 03:16:38 open /bsroot/config/ingress.template: no such file or directory
2016/09/27 03:16:38 open /bsroot/config/skydns.template: no such file or directory
panic: panic: open /bsroot/config/ingress.template: no such file or directoryopen /bsroot/config/skydns.template: no such file or directory
```
bsroot.sh脚本没有把addons目录下的template拷贝到/bsroot/config

--- a/bsroot.sh
+++ b/bsroot.sh
@@ -192,19 +192,6 @@ prepare_cc_server_contents() {
     printf "Generating install.sh ... "
     cat > $BSROOT/html/static/cloud-config/install.sh <<EOF
 #!/bin/bash
-#Obtain devices
-devices=\$(lsblk -l |awk '\$6=="disk"{print \$1}')
-# Zap all devices
-# NOTICE: dd zero to device mbr will not affect parted printed table,
-#         so use parted to remove the part tables
-for d in \$devices
-do
-  for v_partition in \$(parted -s /dev/\${d} print|awk '/^ / {print \$1}')
-  do
-     parted -s /dev/\${d} rm \${v_partition}
-  done
-done
-
 # FIXME: default to install coreos on /dev/sda
 default_iface=\$(awk '\$2 == 00000000 { print \$1  }' /proc/net/route | uniq)
 

--- a/bsroot.sh
+++ b/bsroot.sh
@@ -12,9 +12,13 @@ fi
 realpath() {
     [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
 }
+
 CLOUD_CONFIG_TEMPLATE=$(realpath $(dirname $0)/cloud-config-server/template/cloud-config.template)
+INGRESS_TEMPLATE=$(realpath $(dirname $0)/addons/template/ingress.template)
+SKYDNS_TEMPLATE=$(realpath $(dirname $0)/addons/template/skydns.template)
 CLUSTER_DESC=$(realpath $1)
 SEXTANT_DIR=$(realpath $(dirname $0))
+
 BS_IP=`grep "bootstrapper:" $CLUSTER_DESC | awk '{print $2}' | sed 's/ //g'`
 if [[ "$?" -ne 0 ||  "$BS_IP" == "" ]]; then
     echo "Failed parsing cluster-desc file $CLUSTER_DESC for bootstrapper IP".
@@ -166,6 +170,7 @@ prepare_cc_server_contents() {
     wget --quiet -c -O $BSROOT/html/static/kubelet https://storage.googleapis.com/kubernetes-release/release/$hyperkube_version/bin/linux/amd64/kubelet
     wget --quiet -c -O $BSROOT/kubectl https://storage.googleapis.com/kubernetes-release/release/$hyperkube_version/bin/linux/amd64/kubectl
     chmod +x $BSROOT/html/static/kubelet
+    chmod +x $BSROOT/kubectl
     echo "Done"
 
     # setup-network-environment will fetch the default system IP infomation
@@ -178,10 +183,28 @@ prepare_cc_server_contents() {
     cp $CLOUD_CONFIG_TEMPLATE $BSROOT/config/ || { echo "Failed"; exit 1; }
     cp $CLUSTER_DESC $BSROOT/config/cluster-desc.yml || { echo "Failed"; exit 1; }
     echo "Done"
+       
+    printf "Copying ingress template and skydns template ... "
+    cp $INGRESS_TEMPLATE $BSROOT/config/ || { echo "Failed"; exit 1; }
+    cp $SKYDNS_TEMPLATE $BSROOT/config/ || { echo "Failed"; exit 1; }
+    echo "Done"
 
     printf "Generating install.sh ... "
     cat > $BSROOT/html/static/cloud-config/install.sh <<EOF
 #!/bin/bash
+#Obtain devices
+devices=\$(lsblk -l |awk '\$6=="disk"{print \$1}')
+# Zap all devices
+# NOTICE: dd zero to device mbr will not affect parted printed table,
+#         so use parted to remove the part tables
+for d in \$devices
+do
+  for v_partition in \$(parted -s /dev/\${d} print|awk '/^ / {print \$1}')
+  do
+     parted -s /dev/\${d} rm \${v_partition}
+  done
+done
+
 # FIXME: default to install coreos on /dev/sda
 default_iface=\$(awk '\$2 == 00000000 { print \$1  }' /proc/net/route | uniq)
 
@@ -269,12 +292,6 @@ generate_tls_assets() {
     openssl req -new -key bootstrapper.key -out bootstrapper.csr -subj "/CN=bootstrapper" > /dev/null 2>&1 || { echo "Failed"; exit 1; }
     openssl x509 -req -in bootstrapper.csr -CA ca.pem -CAkey ca-key.pem -CAcreateserial -out bootstrapper.crt -days 365 > /dev/null 2>&1 || { echo "Failed"; exit 1; }
     echo "Done"
-
-    # Note: we need to run the following commands on the bootstrapper server to import ca.crt.
-    #
-    #  mkdir -p /etc/docker/certs.d/$BS_IP:5000
-    #  rm -rf /etc/docker/certs.d/$BS_IP:5000/*
-    #  cp ca.pem /etc/docker/certs.d/$BS_IP:5000/ca.crt
 }
 
 check_prerequisites

--- a/docker/entrypoint.sh
+++ b/docker/entrypoint.sh
@@ -22,9 +22,8 @@ cloud-config-server -addr ":80" \
   -cluster-desc-file /bsroot/config/cluster-desc.yml \
   -cluster-desc-url "" \
   -ca-crt /bsroot/tls/ca.pem \
-  -ca-key /bsroot/tls/ca-key.pem \
-  -ingress-template-file /bsroot/config/ingress.template \
-  -skydns-template-file /bsroot/config/skydns.template &
+  -ca-key /bsroot/tls/ca-key.pem &
+
 # start registry
 registry serve /bsroot/config/registry.yml &
 sleep 2

